packages/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -58,9 +68,24 @@ class GoogleMap extends StatefulWidget {
   /// True if the map view should relay camera move events to Flutter.
   final bool trackCameraPosition;
 
-  // Markers to be placed on the map.
+  /// Markers to be placed on the map.
   final Set<Marker> markers;
 
+  /// Callback for when the camera move started.
```

Comment:
This doesn't add much information.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation

Documentation is definitely hard 😄 I sometimes find the prompts here helpful:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#writing-prompts-for-good-documentation

Some questions I might have been looking for answers in the docs are: is this called on the first time the map is loaded? is this called when the position is updated programmatically?

(similar comments to the docs of the other new fields below)

Reply:
Great read. Will revise the docs based on this.